.github/workflows/bump_version.yml

```
@@ -0,0 +1,150 @@
+name: Bump version
+
+on: 
+  pull_request:
+      branches: [ master ]
+      types: [opened, edited, reopened]
```

```suggestion
      types: 
        - opened
        - edited
        - reopened
        - synchronize
      
```

We could even think about not triggering this workflow on a draft. Then we would need to add `ready_for_review` to the list